solutions/devpsrint-michael-douglas-1/GitHubAppTests/SnapshotTests/LoadingViewTest.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoadingViewTest.swift
+//  GitHubAppTests
+//
+//  Created by Fabio Martinez on 26/01/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import GitHubApp
+
+class LoadingViewTest: XCTestCase {
+
+    let snapshotSize = CGSize(width: 500, height: 200)
+
+    func testLoadingView() {
```

Comment:
Seguindo as boas práticas de nomenclatura de testes, considere seguir este padrão.
```suggestion
    func test_loadingView_whenSnapshot_shouldSucceed() {
```

Reply:
Padronizado nome da func.